tests/JsonTest.php

```diff
@@ -109,13 +111,13 @@ public function testParseErrorDetectMissingColon()
         $this->expectParseException($json, 3);
     }
 
-    public function testParseErrorUtf8()
+    public function testParseErrorUtf8(): void
     {
         $json = "{\"message\": \"\xA4\xA6\xA8\xB4\xB8\xBC\xBD\xBE\"}";
         $this->expectParseException($json, -1, 'Malformed UTF-8 characters, possibly incorrectly encoded', JSON_ERROR_UTF8);
     }
 
-    private function expectParseException($json, $line, $text = null, $code = JSON_ERROR_SYNTAX)
+    private function expectParseException($json, $line, $text = null, $code = JSON_ERROR_SYNTAX): void
     {
         try {
             $result = Json::parse($json);
```

Comment:
This code has been problematic during `PHPUnit testing`. The error messages are as follows:

```
1) Bolt\Common\Tests\JsonTest::testParseErrorObjectEmptyString
TypeError: Argument 1 passed to Bolt\Common\Json::parse() must be of the type string or null, object given, called in /home/travis/build/bolt/common/tests/JsonTest.php on line 123
/home/travis/build/bolt/common/src/Json.php:122
/home/travis/build/bolt/common/tests/JsonTest.php:123
/home/travis/build/bolt/common/tests/JsonTest.php:33
```